addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1287,3 +1287,7 @@ def execute_command_who(self, **kwargs):
             msg = _("Users in this channel: %(members)s %(dots)s and you.", members=", ".join(members), dots=dots)
 
         self._send_transient_message(self.env.user.partner_id, msg)
+
+    @api.model
+    def _get_thread_access_to_post(self):
```

Comment:
As told, this is a duplicate of ``_mail_post_access``, we can use existing code :)